examples/user-list/index.html

```diff
@@ -0,0 +1,42 @@
+<user-list>
+  <h1>User list:</h1>
+
+  <button>Load users</button>
+
+  <ul>
+    <template name=users>
+      <li>{name}
+    </template>
+  </ul>
+
+</user-list>
+
+<script src=user-list/rxr.js></script>
+<script src=//unpkg.com/rxjs/bundles/Rx.min.js></script>
+
+<script src=/snuggsi.es></script>
+<script src=user-list/reactive.es></script>
+
+<script defer>
+
+Element `user-list`
+
+(class extends Reactive ( HTMLElement ) {
+
+  onclick ()
+    { this.actions.onclick () }
+
+  onstatechange (state) {
```

Comment:
Perhaps `onstatechange` should go in `reactive.js` Unless we want to provide this API to the end component author. Keeping this handler here doesn't feel very _"batteries included"_  As @brandondees would say.